packages/shared/src/components/search/SearchPanel/SearchPanelInput.tsx

```diff
@@ -68,14 +70,12 @@ export const SearchPanelInput = ({
     onFocus: externalOnFocus,
     onBlur: externalOnBlur,
     onClick: externalOnClick,
-    placeholder = searchPanel.isActive
-      ? 'Search posts or ask a question...'
-      : 'Search',
```

Comment:
Let's make sure here they want to experiment with both of these versions 🙏

Reply:
> Let's make sure here they want to experiment with both of these versions 🙏

You mean you want the experiment to only be displayed when the search panel is "active"? 😄

Reply:
Well, yeah that's what we need to check with product in this case, highlight to them that in the meantime we have 2 placeholders for when the panel is active and when panel is not active. 
Just because with the changes we're doing now in this PR, we're defaulting to only one placeholder for both.

Reply:
As for where to check this with product, u can either do on product-chatter channel, or in the experiment approval thread you added in experiments channel by tagging @product